src/main/java/edu/ucsb/cs156/happiercows/entities/UserCommons.java

```diff
@@ -28,5 +28,13 @@
 
   private int totalWealth;
 
+    public static UserCommons dummyUserCommons(long id) {
```

Comment:
Should this live in the main codebase? I think it makes more sense to move this somewhere under the test directory to avoid cluttering the main file and to prevent the method from accidentally being used in production code.

Reply:
refactored into a method in the test file